src/common/helpers/proposalTitleAndDescription/ancillaryData.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from "ethers";
+
+export function decodeAncillaryDataHexString(ancillaryDataHexString: string) {
```

Comment:
I've extracted this logic to a helper function which we can use elsewhere going forward.